### ISSUE TITLE
game crash when looking at mod in JEI or creative mode with appleskin

### ISSUE THREAD

**Issue by DIRECTVIRUS** [DakotaPride/create-garnished#13]
when looking at the mod in the creative inventory or looking at an item from the mod in JEI the game crashes.

happens with the appleskin mod

**Comment by DakotaPride** [comment]
Hello! Apologies for the long wait, some other information may help with this issue:

What MC Version are you using?

What Garnished Version are you using?

Are you using Fabric, Forge, Quilt, etc.?

This was an issue in a previous version, so this information is crucial in order to fix the issue.

**Comment by DIRECTVIRUS** [comment]
ahh my apologies for not including in the original issue.

1.19.2 (game version)

43.2.23 (forge version)

garnished version: 0.8-1.19.2 (was the version my launcher grabbed as latest)

**Comment by DakotaPride** [comment]
One more thing, is it just in general, or is it when you are hovering over a specific food in particular?

**Comment by DIRECTVIRUS** [comment]
it happens in general whenever a create garnished item appears. not even hovering over the items

e.g. I want a piston. I search pis

the pistachio nut variants appear and the game freezes before crashing

happens with any variation of one of the create garnished nuts appearing in a GUI

have not tried giving myself one via command.

**Comment by DakotaPride** [comment]
Terribly sorry for the problem, but could you provide a recording/recreation of this happening? Testing in an instance with the mods listed does not have the same issue occur.

Apple skin version utilized: 2.4.2 for 1.19.2
JEI version utilized: 11.6.0.1018 for 1.19.2

**Comment by DIRECTVIRUS** [comment]
it's a custom pack but i'll provide the latest.log of the crash and a video.

looking at latest.log the appleskin class is what is having the issue (at least that is what the CF discord helpers spotted)
[latest.log](https://github.com/DakotaPride/create-garnished/files/12752046/latest.log)

okay so I uninstalled the mod and re-installed it and now the crash does not happen as it used to-

I am unable to reproduce it. I removed the mod so I could play and then readded it to test and it does not occur.
